packages/quakespasm.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Quakespasm < Package
+  description 'A modern, cross-platform Quake game engine based on FitzQuake.'
+  homepage 'http://quakespasm.sourceforge.net'
+  version '0.93.1'
+  source_url 'https://sourceforge.net/projects/quakespasm/files/Source/quakespasm-0.93.1.tgz'
```

Comment:
Please use this instead:
`source_url 'https://prdownloads.sourceforge.net/project/quakespasm/Source/quakespasm-0.93.1.tgz'`
This will prevent the delay when downloading.